### PR TITLE
Add calculateFees function

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mycrypto/gas-estimation",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (Mrtenz) [inline package.json]: Should be minor bump

File: README.md [hunk 2]
```diff
@@ -31,13 +31,19 @@ console.log(maxFeePerGas, maxPriorityFeePerGas);
 
 ## API
 
-The library exposes a single function to estimate gas fees based on the latest 10 blocks.
+The library exposes a function to estimate gas fees based on the latest 10 blocks and the underlying function used to calculate the estimate.
```
Comment (Mrtenz) [inline README.md]:
```suggestion
The library exposes a function to estimate gas fees based on the latest 10 blocks, and the underlying function used to calculate the estimate.
```

File: src/eip1559.ts [hunk 3]
```diff
@@ -125,3 +101,30 @@ export const estimateFees = async (provider: ProviderLike): Promise<EstimationRe
...
+      baseFee >= PRIORITY_FEE_ESTIMATION_TRIGGER
+        ? await getFeeHistory(provider, hexlify(FEE_HISTORY_BLOCKS), hexlify(blockNumber), [
+            FEE_HISTORY_PERCENTILE
+          ])
+        : undefined;
+
+    return calculateFees(baseFee, feeHistory);
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.error(err);
```
Comment (Mrtenz) [inline src/eip1559.ts:127]: I know this is not fully related to this PR, but logging errors from within a library seems kind of like a bad practice to me

**Review by Mrtenz — CHANGES_REQUESTED** [review]

File: src/eip1559.ts (hunk 3 above)
Comment (FrederikBolding) [reply src/eip1559.ts:127]: Yeah, maybe for next version we figure out a better solution?

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**